### ISSUE TITLE
Support performance of HTML files in all JavaScript front ends

### ISSUE THREAD

**Issue by gogins** [csound/csound#552]
Currently the JavaScript front ends (CsoundQt, csound.node, Csound for Android, and Csound for PNaCl) are not consistent. Some perform HTML files with embedded Csound code only, some perform only CSD files with embedded <html> only, and at this time only csound.node performs either.

This request is to support HTML files with embedded Csound code in all JavaScript front ends. This means adding such functionality to CsoundQt and Csound for Android.

**Comment by gogins** [comment]
I began to implement this and ran into significant obstalces. The main problem is that the design of JCsoundObj in the CsoundApplication assumes that an initial CSD file that sets the "-o" option will be used. In my implementation, I examined the filename and if it ends in "html", csoundStart(csd) is called, but instead csound.Start() is called. The problem with this is that the default "-otest.wav" option is automtically used, and this fails on Android, plus it is not always the desired behavior.
